src/com/amazon/ionschema/internal/constraint/TimestampOffset.kt

```diff
@@ -43,6 +43,7 @@ internal class TimestampOffset(
      * exactly with values returned by IonTimestamp.localOffset.
      */
     private val validOffsets: Set<Int?>
+    private val negateOffsets: Boolean
```

Comment:
Because the offsets are signed integers, the word `negate` is a bit misleading. How about `offsetsAreValid`, indicating the contents of the `offsets` variable above?